index.js

```diff
@@ -131,8 +142,16 @@ module.exports = function(app) {
         method: zone.method,
         timestamp: (new Date()).toISOString()
       }
+    } else {
+      // Default to "normal" zone
+      value = {
+        state: "normal",
+        message: "Value is within normal range",
+        method: [],
+        timestamp: (new Date()).toISOString()
```

Comment:
Let's remove the timestamp from the value, the server will assign timestamp for the delta. Again this is old code.

Reply:
Ok